packages/engine/src/worker/runner/python/runner.py

```diff
@@ -166,6 +166,10 @@ def run(self):
                     self.kill()
                     break
 
+
+                continue # TODO: Don't ignore messages (after fixing message handling).
```

Comment:
I wonder why clippy isn't complaining for unreachable code or something like that

Reply:
LGTM analysis is.

> This pull request **introduces 1 alert** when merging [03b1c4d](https://github.com/hashintel/hash/commit/03b1c4d49342e8c7a287ebb5cb1c8bb63e3bcb96) into [11d2945](https://github.com/hashintel/hash/commit/11d29451492f5ccbb91dfb20d6f2d194609b1471) - [view on LGTM.com](https://lgtm.com/projects/g/hashintel/hash/rev/pr-192b632e841ea2e6d176a67a82d3fd000c2a28f1)
> 
> **new alerts:**
> 
> * 1 for Unreachable code

Reply:
It's Python, so LGTM analysis is complaining :)